### PR TITLE
Player Class

### PR DESCRIPTION
**Pull request by ducampcarlos**

### CODE REVIEW

File: src/Player.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	 */
+	
+	//Attributes
+	private ABML<item> inventory; //Collections of the items that the player has.
+	private PlayableCharacter team; //Character that the Player will use.
+	//private ABML<PlayableCharacter team; //If there's time, there'll be a Collections of characters that the Player could use.
+	private int gold; //Amount of gold the Player will have
+	
+	//Constructor
+	public Player() {}
```
Comment (rightonhana) [inline src/Player.java]: Set attributes values in the constructor

File: src/Player.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	
+	//Getters and Setters
+	public ABML<item> getInventory() {
+		return inventory;
+	}
+
+	public void setInventory(ABML<item> inventory) {
+		this.inventory = inventory;
+	}
+
```
Comment (AlejandroWoloch) [inline src/Player.java:32]: I think you shouldn't send as parameter a full list of thing to set, so inventory shouldn't have a setter, but methods to add and remove single items to the inventory

File: src/Player.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	}
+	
+	public boolean equals(Object obj) {
+		if (obj != null && obj.getClass().isInstance(Player.class)) {
+			return ((User) obj).getId() == this.getId();
+		} else {
+			return false;
+		}
+	}
+
```
Comment (AlejandroWoloch) [inline src/Player.java:84]: If User class already has an equals method defined by its ID, you could just do => return super.equals(obj);

Comment (ducampcarlos) [reply src/Player.java:84]: Fixed, thank you

File: src/Player.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	}
+	
+	//public void administrateInventory() {} //Not going to use this for now
+
+
+	//To String, Equals, Hashcode.
+	
+	@Override
+	public String toString() {
+		String message= "/nPlayer [Character name: %s, Amount of Gold: %d";
```
Comment (rightonhana) [inline src/Player.java]: Add the last square bracket

**Comment by rightonhana** [comment]
Looks good to me!

**Pull request merged**